<launch>
    <include file="$(find igvc)/launch/localization.launch" />

    <!-- Map Node -->
    <node name="new_mapper" pkg="igvc" type="new_mapper" output="screen">
            <param name="topics" type="string" value="/scan/pointcloud /usb_cam_center/line_cloud /pothole_cloud"/>
            <param name="occupancy_grid_length" type="int" value="200" />
            <param name="occupancy_grid_width" type="int" value="200" />
            <param name="occupancy_grid_resolution" type="double" value="0.1" />
            <param name="start_X" type="double" value="50"/>
            <param name="start_Y" type="double" value="10"/>
            <param name="orientation" type="double" value="0.0"/>
            <param name="debug" type="bool" value="true"/>

    </node>

</launch>
